main.go

```diff
@@ -25,6 +25,7 @@ processors:
 
 exporters:
   desktop:
+    endpoint: localhost:8880
```

Comment:
We probably don't want to change away from `localhost:8000` as the default

Reply:
Oh oops, this is an artefact of me playing with it. I will put it back.